apisix/cli/ops.lua

```diff
@@ -916,10 +916,15 @@ local function stop(env)
 end
 
 
+local function sleep(n)
+  execute("sleep " .. tonumber(n))
+end
+
 local function restart(env)
   -- test configuration
   test(env)
   stop(env)
+  sleep(1)
```

Comment:
I think just sleeping 1 second is not enough, better adding explicit log message
```
--- a/apisix/cli/ops.lua
+++ b/apisix/cli/ops.lua
@@ -805,8 +805,8 @@ local function start(env, ...)
 
         local ok, err, err_no = signal.kill(pid, signone)
         if ok then
-            print("APISIX is running...")
-            return
+            print("the old process is still running, the new one won't be started!")
+            return
         -- no such process
         elseif err_no ~= errno.ESRCH then
             print(err)
```

Reply:
you can search the code `APISIX is running`, and replace them all with your new log

Reply:
ok , i fix it